RiotSharp/Endpoints/StaticDataEndpoint/ReforgedRune/Cache/ReforgedRunePathListStaticWrapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using RiotSharp.Misc;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RiotSharp.Endpoints.StaticDataEndpoint.ReforgedRune.Cache
+{
+    class ReforgedRunePathListStaticWrapper
+    {
+        public Language Language { get; set; }
+        public string Version { get; set; }
+
+        public List<ReforgedRunePathStatic> ReforgedRunePaths { get; set; }
+
+        public ReforgedRunePathListStaticWrapper(Language language, string version, List<ReforgedRunePathStatic> reforgedRunePaths)
+        {
+            Language = language;
+            Version = version;
+            ReforgedRunePaths = reforgedRunePaths;
+        }
+
+        public bool Validate(Language language, string version)
```

Comment:
I think it'd be nice to have this up in an interface, wdyt? Not for this pr of course